tests/lib/app_keys.py

```diff
@@ -0,0 +1,39 @@
+
+def assert_api_product_in_application(developer_app_keys_api, dev_email, app_name, key_id, api_product_name):
+    r = developer_app_keys_api.get_app_key(
```

Comment:
what is `r`?

Reply:
It's just a variable to denote whatever response we get back from `.get_app_key`, it's short here because the scope is very short so it's easy to see where it's being used. i.e. It's not likely you'll have to ever scroll up to see what `r` is because this function almost always fits on one screen. I can change it to `response` if needed, but this doesn't do a lot to help a reader understand the shape of that variable. One still has to go and look at the docs for `.get_app_key`, which in turn is not very helpful and takes you to the apigee docs which are a nightmare.

One could change this to annotate what we're pulling out fo the response better. And call it `current_api_products_in_app_key = developer_app_keys_api.get_app_key(...)['apiProducts']`. I thought it would be handy for this function to return its response should somebody want to use other details from that response. Shall I implement this suggestion and sacrifice the full returned body?